example/gluon/style_transfer/main.py

```diff
@@ -0,0 +1,214 @@
+import time
+import random
+import os
+import mxnet as mx
+import numpy as np
+np.set_printoptions(precision=2)
+from PIL import Image
+
+from mxnet import gluon
+from mxnet.gluon import nn, autograd, Block, HybridBlock, Parameter, ParameterDict
+import mxnet.ndarray as F
+
+import net
+import utils
+from option import Options
+
+def train(args):
+    # temporary solutution for dataloader
+    from torchvision import datasets
+    from torchvision import transforms
```

Comment:
What's missing in gluon that requires using torchvision? Asking mxnet users to install torchvision might not be a good idea.